xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
+     contain security issues than console programs.
+    </para>
+
+    <para>
+     To deny the &rootuser; user login to graphical sessions you
+     can apply the same basic steps as outlined in
+     <xref linkend="sec-sec-prot-restrict-root-tty"/>.
+     You only need to also add the <literal>pam_securetty</literal> module to
+     the PAM stack file belonging to the display manager, for example
+     <filename>/etc/pam.d/gdm</filename> for GDM. The graphical session also
+     runs on a TTY device, <literal>tty7</literal>, by default. Therefore,
+     when you restrict &rootuser; logins to
+     <literal>tty1</literal>, then a &rootuser; login in the
+     graphical session will be denied.
+    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-sec-prot-restrict-root-ssh">
+    <title>Restricting SSH Logins</title>
+    <para>
+     By default the &rootuser; user is also allowed to login
+     into a machine remotely via the SSH network protocol (if the SSH port is
+     not blocked by the firewall). To restrict this possibility the following
+     change of the OpenSSH configuration needs to be performed:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>Edit <filename>/etc/ssh/sshd_config</filename> and adjust the following parameter:</para>
+      <screen>PermitRootLogin no</screen>
+     </listitem>
+     <listitem>
+      <para>Restart the <literal>sshd</literal> service to make the changes effective:</para>
+      <screen>systemctl restart sshd.service</screen>
+     </listitem>
+    </itemizedlist>
+    <note>
+     <para>
+      Using the PAM module <literal>pam_securetty</literal> is not suitable in
+      case of OpenSSH, because SSH logins not necessarily go through the PAM
+      stack during the auth phase (e.g. when using SSH public key authentication).
+      Also an attacker could be able to differentiate between a wrong password
+      and a successful login that was only rejected later on by policy.
+     </para>
+    </note>
+   </sect2>
+
+  </sect1>
+  <sect1 xml:id="sec-sec-prot-inactivity-logout">
+   <title>Setting an Inactivity Timeout for Interactive Shell Sessions</title>
+   <para>
+    It can be a good idea to terminate an interactive shell session after a
+    certain time of inactivity. For example an administrator might have
+    forgotten to terminate a local login session on a sensitive system. This
+    could provide an attack vector if a local attacker finds such a session
+    unguarded. Another effect of idle shell sessions is waste of system
+    resources.
+   </para>
+   <para>
+    By default there is no inactivity timeout for shells, so nothing will
+    happen if a shell stays open unused for days or even years. It is possible
+    to configure most shells in a way that idle sessions terminate
+    automatically after a certain amount of time. In the following example we
+    will show how to set an inactivity timeout for a number of
```

Comment:
To avoid "we" and to make it consistent with the other text:

```suggestion
    automatically after a certain amount of time. The following example
    shows how to set an inactivity timeout for a number of
```